pkgs/top-level/python-packages.nix

```diff
@@ -11692,6 +11694,7 @@ self: super: with self; {
 
   tensorflowWithCuda = self.tensorflow.override {
     cudaSupport = true;
+    cudaPackages = pkgs.cudaPackages_11_7;
```

Comment:
This way `(import nixpkgs { config.cudaSupport = true; }).tensorflow` still uses the default `cudaPackages`. I think we'd want to `tensorflow = self.tensorflow.override { cudaPackages = ...; }` instead.